### ISSUE TITLE
autoviz `getRegistry` of null error

### ISSUE THREAD

**Issue by brollb** [dfst/deepforge#179]
When editing operation (changing interface) and then moving to the parent the following error is thrown:

```
TypeError: Cannot read property 'getRegistry' of null    AutoVizPanel.js: 77
```

**Comment by brollb** [comment]
https://github.com/dfst/webgme-autoviz/issues/6

**Comment by brollb** [comment]
Fixed in `webgme-autoviz` v2.0.2
